### PR TITLE
Add ability to copy a table to the same database under a new name

### PR DESCRIPTION
**Pull request by funkswing**
Allow `thinker sync` to copy an existing table to a new table name on the same DB 🔥🔥🔥 
Also added changefeed code for maintaining the changes during the sync.

### CODE REVIEW

File: lib/commands/sync.js [hunk 1]
```diff
@@ -49,8 +49,19 @@ let HELPTEXT = `
...
                                               3 = No logs
 
+
+    ██████╗  █████╗ ████████╗██╗  ██╗███████╗██████╗ 
+    ██╔════╝ ██╔══██╗╚══██╔══╝██║  ██║██╔════╝██╔══██╗
+    ██║  ███╗███████║   ██║   ███████║█████╗  ██████╔╝
+    ██║   ██║██╔══██║   ██║   ██╔══██║██╔══╝  ██╔══██╗
+    ╚██████╔╝██║  ██║   ██║   ██║  ██║███████╗██║  ██║
+      ╚═════╝ ╚═╝  ╚═╝   ╚═╝   ╚═╝  ╚═╝╚══════╝╚═╝  ╚═╝
+
```
Comment (claudiordgz) [inline lib/commands/sync.js:59]: rainbow-eyes

Also we might want to talk about making this repo private

File: lib/main.js [hunk 2]
```diff
@@ -30,7 +30,11 @@ module.exports = function (argv) {
     let command = _.first(argv['_'])
     argv['_'] = argv['_'].slice(1)
     if (commands[command]) {
-      yield commands[command](argv)
+      if (command === "clone") {
+        console.log(`"clone" is invalid command for Gather. Use "sync"`)
```
Comment (claudiordgz) [inline lib/main.js:33]: We could also delete the `clone.js` file completely

**Review by claudiordgz — APPROVED** [review]
Can I ECSify this?

File: lib/commands/sync.js (hunk 1 above)
Comment (funkswing) [reply lib/commands/sync.js:59]: _THE WORLD MUST KNOW_

File: lib/main.js (hunk 2 above)
Comment (funkswing) [reply lib/main.js:33]: YOu already did...

File: scripts/changefeeds/consume.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    if (data.type === "add" && data.new_val) {
+        try {
+            await r.table("files").insert(data.new_val);
+            conn.ack(msg);
+        } catch(e) {
+            conn.nack(msg, undefined, false)  // nack({ requeue: false })
+        }
+    } else if (data.type === "change") {
+        try {
+            await r.table("files").replace(data.new_val);
```
Comment (calexander3) [inline scripts/changefeeds/consume.js]: Do you need to get before replace?

**Review by calexander3 — APPROVED** [review]

File: lib/commands/sync.js [hunk 4]
```diff
@@ -93,6 +104,43 @@ function getByCompany(connection, database, table, companyIds) {
...
+    // return doc.hasFields("settings").not()
+    // .or(
+    //   doc("settings").hasFields("automatic").not()
+    //   )
+    // })
+    .map(function (row) { return {id: row('id'), hash: connection.uuid(row.toJSON())} })
+    .run({ cursor: true })
+}
+
+/** Gets a DB cursor to the items filtered by Secondary Index Company
```
Comment (masondaven) [inline lib/commands/sync.js:125]: Do these need to be updated to be the correct params?

**Pull request merged**